federatedscope/cl/dataloader/Cifar10.py

```diff
@@ -0,0 +1,190 @@
+import math
```

Comment:
To reproduce baseline, the `sample_splitter` is needed.
https://github.com/akhilmathurs/orchestra/blob/228d7a6379b6788e7dc288d3a9557d62b940c47a/utils.py#L44